### PR TITLE
Items added to list

### PR DESCRIPTION
**Pull request by PierceMcGeough**
Beanstalkapp to Source Code Repos
Sublime Text to IDE and Code Editors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -356,6 +357,7 @@ Table of Contents
    * https://www.visualstudio.com/en-us/products/visual-studio-community-vs.aspx - Visual Studio. Not only for Windows and .NET
    * https://codio.com/ - Codio is a cloud-based computer programming platform for universities, schools, and developer professionals.
    * http://www.stackhive.com/ - Cloud based IDE in browser that supports HTML5/CSS3/jQuery/Bootstrap
+   * http://www.sublimetext.com/3 - Sublime Text is a sophisticated text editor for code, markup and prose.
```
Comment (ripienaar) [inline README.md]: i believe the other here are hosted or very tightly coupled with a hosted component, sublime is mainly a download standalone app am I right?

If that's the case I'd rather not list it

**Comment by PierceMcGeough** [comment]
Yes it is downloadable but has many plugins that can be installed to make life easier. Jetbrains PHPStorm is downloadable and standalone and it is on the list. We all use both at work PHPStorm for backend and Sublime for the front.

**Comment by ripienaar** [comment]
I'll rather remove phpstorm then, this list is for *aaS only as per the top of the readme and description

**Comment by PierceMcGeough** [comment]
Fair enough then.

One other thing
https://www.atlassian.com/opensource/overview  should really be
https://www.atlassian.com/software/jira

I think first link is a bit misleading. Need to fish around to get to the Jira page from there

**Comment by ripienaar** [comment]
at the time thought the current link was easier to find the open source offer but yeah just linking to https://www.atlassian.com/ would be best since they give all their software away for free for OSS

**Pull request closed without merging**